docs/guides/client.md

```diff
@@ -0,0 +1,83 @@
+# Prism Client
+
+Prism includes an full featured HTTP Client that you can use to seamlessly perform requests to both a real server and a mocked document. The client is modeled after Axios so it may feel familiar.
```

Comment:
`a full
```suggestion
Prism includes a fully-featured HTTP Client that you can use to seamlessly perform requests to both a real server and a mocked document. The client is modeled after Axios so it may feel familiar.
```